mcs/class/System/Microsoft.CSharp/CSharpCodeGenerator.cs

```diff
@@ -93,18 +93,16 @@ private CompilerResults FromFileBatch (CompilerParameters options, string[] file
 
 			mcs.StartInfo.CreateNoWindow=true;
 			mcs.StartInfo.UseShellExecute=false;
-			mcs.StartInfo.RedirectStandardOutput=true;
```

Comment:
We should keep this and instead do the same what we do for stderr (and just discard the data).

Reply:
Yes, that was my first impression too, but then the code was ignoring `stdout` and I don't really know what would/should we do with it? Just discard?